drivers/storage/portworx/component/autopilot.go

```diff
@@ -205,11 +205,11 @@ func (c *autopilot) createConfigMap(
 			if key == AutopilotDefaultReviewersKey {
 				params += fmt.Sprintf(`
     %s:`, AutopilotDefaultReviewersKey)
-				values := strings.Split(val, ",")
-				for _, v := range values {
-					v = strings.TrimSpace(v)
+				var reviewers []interface{} = val.([]interface{})
+				for _, r := range reviewers {
+					r = strings.TrimSpace(r.(string))
```

Comment:
Could we please add some type-checking here?  I.e.

```go
s := r.(string)       // this throws PANIC if r was not string-type
s, ok := r.(string)   // always works, will set `ok = false` if r was not string-type
```

So, the current code is susceptible to "panic's"